templates/openedx_nodebb_discussion/nodebb.html

```diff
@@ -0,0 +1,23 @@
+## mako
+<%! from django.utils.translation import ugettext as _ %>
+<%namespace name='static' file='/static_content.html'/>
+<%inherit file="/main.html" />
+<%block name="bodyclass">view-openedx_nodebb_discussion is-in-course course</%block>
+<%block name="pagetitle">${_("NodeBB Discussion")}</%block>
+<%block name="headextra">
+<%static:css group='style-course'/>
+</%block>
+<%include file="/courseware/course_navigation.html" args="active_page='openedx_nodebb_discussion'" />
+
+<div class="discussion discussion-board page-content-container" id="discussion-container">
+    <div class="page-content">
+       <iframe width="100%" height="500"
+            src="${nodebb_url}\embed?category_id=${category_id}"
```

Comment:
There is a `\` in after the `{nodebb_url}`, is it intentional?

Reply:
It should be `/` btw

Reply:
done.

Reply:
@asamolion it was added mistakenly.